sonaranalyzer-dotnet/its/expected/Nancy/Nancy.Demo.Authentication.Forms.TestingDemo-{948A8EF6-D50C-45EA-9AFD-7A4723ADAB0B}-S1075.json

```diff
@@ -1,30 +0,0 @@
-{
```

Comment:
The _TestingDemo_ project references the legacy XUnit assembly so it is now treated as a test project, so the set of rules that is run is different.

The issues that are no longer being produced are all for _Main_ rules.